macOS/UnitTests/WebsiteBreakageReport/WebsiteBreakageReportTests.swift

```diff
@@ -146,6 +148,9 @@ class WebsiteBreakageReportTests: XCTestCase {
         XCTAssertEqual(queryItems[valueFor: "manufacturer"], "Apple")
         XCTAssertEqual(queryItems[valueFor: "os"], "12")
         XCTAssertEqual(queryItems[valueFor: "gpc"], "true")
+        XCTAssertEqual(queryItems[valueFor: "consentManaged"], "true")
+        XCTAssertEqual(queryItems[valueFor: "consentOptoutFailed"], "true")
+        XCTAssertEqual(queryItems[valueFor: "consentSelftestFailed"], "true")
```

Comment:
as discussed in MM, let's change the actual query values to "1" and "0", to make it consistent with other platforms